dynamodb/dynamo_lock_table.go

```diff
@@ -204,3 +215,53 @@ type TableDoesNotExist struct {
 func (err TableDoesNotExist) Error() string {
 	return fmt.Sprintf("Table %s does not exist in DynamoDB! Original error from AWS: %v", err.TableName, err.Underlying)
 }
+
+// Encrypt the TFState Lock table - If Necessary
+func UpdateLockTableSetSSEncryptionOnIfNecessary(tableName string, client *dynamodb.DynamoDB, terragruntOptions *options.TerragruntOptions) error {
+	tableSSEncrypted, err := LockTableCheckSSEncryptionIsOn(tableName, client)
+	if err != nil {
+		return err
+	}
+
+	if !tableSSEncrypted {
```

Comment:
To reduce indentation of all of this code, just invert this check:

```go
if tableSSEncrypted {
  terragruntOptions.Logger.Printf("Table %s already has encryption enabled", tableName)
  return nil
}
```

Reply:
Done.